src/components/ui/ImagePass.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import Image from "next/image";
+import Link from "next/link";
+
+interface ImagePassProps {
+  display?: boolean;
```

Comment:
Looks like displaying image is a default behaviour. So probably better to change it to 'hideImage' or 'shouldHideImage' and the default value will be false.